R/hdf5.R

```diff
@@ -98,7 +103,7 @@ print_metadata <- function(metadata, prefix="") {
 write_clusters <- function(f, clusters) {
   clusters_group <- f$create_group("clusters")
 
-  for (i in 1:length(clusters)) {
+  for (i in seq_along(clusters)) {
```

Comment:
Using `seq_along` supports iteration when the list is empty.  The previous code would iterate from [1, 0] if the length of the list was empty.